php/libraries/NDB_BVL_Instrument_JSON.class.inc

```diff
@@ -0,0 +1,289 @@
+<?php
+/**
+ * This file contains the JSON Instrument class
+ *
+ * PHP Version 5
+ *
+ * @category Main
+ * @package  Behavioural
+ * @author   Unknown <example@example.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris-Trunk/
+ */
+namespace Loris\Behavioural;
+use \Exception;
```

Comment:
`\Exception` isn't used, and I don't see why Exception is imported from the root namespace but the other things that are used from it aren't.. I'd just get rid of this line.